tests/app/data/test_schema_validation.py

```diff
@@ -67,21 +67,31 @@ def validate_schema(self, schema, file, json_to_validate):
 
         return errors
 
-    def test_no_duplicate_ids_in_schema(self):
+    def test_no_duplicates_in_schema(self):
         schema_files = self.all_schema_files()
 
         # Certain keys need to be ignored to avoid false positives.
         ignored_keys = ['routing_rules', 'skip_condition']
+        special_key = ['id', 'alias']
 
         for schema_file in schema_files:
+
             unique_id = []
+            unique_alias = []
             with open(schema_file, encoding="utf8") as file:
                 schema_json = load(file)
-                for id_value in self._parse_id_values(schema_json, ignored_keys):
-                    if id_value in unique_id:
-                        self.fail('Duplicate Id found. schema: %s, id: %s' % (schema_file, id_value))
-                    else:
-                        unique_id.append(id_value)
+                for key in special_key:
+                    for value in self._parse_values(schema_json, ignored_keys, key):
+                        if key == 'id':
+                            if value in unique_id:
+                                self.fail('Duplicate ID found. schema: %s, id: %s' % (schema_file, value))
+                            else:
+                                unique_id.append(value)
+                        elif key == 'alias':
+                            if value in unique_alias:
+                                self.fail('Duplicate alias found. schema: %s, alias: %s' % (schema_file, value))
+                            else:
+                                unique_alias.append(value)
 
```

Comment:
I think checking for duplicate Ids and duplicate aliases should be two separate tests. Just makes it clearer when one of them fails, you could re-use much of the logic here by extracting a method and passing 'id' or 'alias' as a parameter.